plugins/katello/3.16/release_notes/release_notes.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Release Notes
+---
+
+# Katello {{ page.version }} Release Notes
+
+For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-3.16/CHANGELOG.md).
+
+## Features
+
+* Support for Pulpcore 3.3
+* Pulp 2 to Pulp 3 File and Docker improvements
+* Qpid replaced with ActiveMQ Artemis for messaging between Candlepin and Katello
+* "Subsciptions expiring soon" email notification customization options
```

Comment:
The whole email is new so this could be 'Subscription expiration notification emails'